analysis/HCP_smoothing/HCP_smoothing_by_task.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+#SBATCH -p gablab
+#SBATCH --time=2:00:00
+#SBATCH --mem=1GB
+#SBATCH --cpus-per-task=2
+#SBATCH -J HCP_smoothing_by_task
+#SBATCH -o log/%x-%A-%a.out
+#SBATCH -e log/%x-%A-%a.err
+#SBATCH --mail-user=dclb@mit.edu
+#SBATCH --mail-type=ALL
+#SBATCH -x node[054-060,100-115]
+
+
+module load openmind/hcp-workbench/1.2.3
+
+task=$1 #read in task to be smoothed
+run=$2 #read in run of the task to be smoothed
+
+WORKPATH="../../../derivatives"
+
+OUTPATH="../../../derivatives/HCP_smoothing"
+mkdir -p $OUTPATH
+OUTPATH=`realpath $OUTPATH`
+
+cd $WORKPATH
+
+for ses in * ;
+do
+    [[ $ses != ses-* ]] && continue
+    echo "smoothing cifti files from $ses"
+    cd $ses
+    for sub in * ;
+    do
+        input_cifti_starred_path="./$sub/$ses/func/*task-$task*run-$run*dtseries.nii"
+        [[ $sub == *.html ]] && continue
+        if [ ! -f $input_cifti_starred_path ]; then
+            echo "file $input_cifti_starred_path not found!"
+            continue
+        fi
+        echo "mkdir -p $OUTPATH/$sub/$ses"
+        mkdir -p "$OUTPATH/$sub/$ses"
+        input_cifti_path=`realpath $input_cifti_starred_path`
+        input_cifti=(${input_cifti_path//// })
+        input_cifti=${input_cifti[12]}
+        output_cifti=smoothed_$input_cifti
+        output_cifti_path=$OUTPATH/$sub/$ses/$output_cifti
+        echo "running wb_command -cifti-smoothing"
+        echo "input $input_cifti_path"
+        echo "output $output_cifti_path"
+        now=$(date)
+        echo "$now"
+        #command inputs in order: cifti file; surface kernel size in terms of sigma, volume kernel size in terms of sigma; direction to smooth along: COLUMN is used for dtseries files, while ROW would be used for dconn files; cifti output file; left surface file; right surface file
+        wb_command -cifti-smoothing \
+        $input_cifti_path \
+        4 4 \
```

Comment:
especially large for the volumetric structures.

Reply:
that's a good point. I was using 4 4 because Jeff and Gasser were using it (I think we all found an online example using this setting and just kept using it). for the subcortical volumetric structures I will change this to fwhm=6 (that's what i use in the volumetric analysis). what do you suggest for the vertex/surface data?

Reply:
will use --fwhm 4 4

Reply:
```suggestion
       -fwhm 4 4 \
```